util/configure

```diff
@@ -370,7 +366,7 @@ push @make_install_cmds,
 
 if ($platform ne 'msys') {
     push @make_install_cmds,
-        "ln -sf $ngx_sbin \$(DESTDIR)$prefix/bin/openresty";
+        "ln -sf \$(DESTDIR)$ngx_sbin \$(DESTDIR)$prefix/bin/openresty";
```

Comment:
This is wrong. It will create a symlink pointing to the wrong location.

Reply:
I guess it depends on where you want to have it link to, but if you have DESTDIR defined, then the symlink will link to the location without $(DESTDIR) added.  All other files are installed with $(DESTDIR) prefixed, so this probably isn't what you want.  Nginx is installed to the prefixed location.  Adding $(DESTDIR) to the $ngx_sbin path makes it point to the nginx binary in the same root folder in all situations.

There is also another problem if you have DESTDIR set.  That is that because the rpath and link locations are defined statically, nginx doesn't find libluajit, unless you specify it in the environment.  IMHO it would be better to have relative symlinks and relative paths set.  I was going to make the necessary changes to make them relative paths, so everything worked as expected if you move the output dir and submit them too.  It makes more sense to me to be able to move the whole folder and for everything still to work, but let me know if that's not what you want.

Reply:
@mclyne The `DESTDIR` variable is usually used for rpm/ubuntu packaging tools to install the software to a temporary location other than under the real root directory. So it is not meant to be the final location for the installed packages. That's why I said if you add `$(DESTDIR)` to the symlink target location, it would be completely wrong (the DESTDIR location won't even exist in the final deployment machine).

Reply:
@mclyne Seems like you confused DESTDIR with PREFIX. They are for completely different purposes.

Reply:
@mclyne So your change above would lead to broken symlinks installed to the final machines after installed the deb/rpm packages.

Reply:
I wasn't confusing DESTDIR with PREFIX, however I wasn't thinking of it being used in that way.  Thanks for the explanation.